controllers/v1/entities.js

```diff
@@ -257,6 +257,66 @@ module.exports = class Entities extends Abstract {
       })
     }
 
+     /**
+  * @api {get} /kendra/api/v1/entities/list
+  * List entities.
+  * @apiVersion 1.0.0
+  * @apiName List entities
+  * @apiGroup Entities
+  * @apiSampleRequest /kendra/api/v1/entities/list
+  * @param {json} Request-Body:
+  * {
+  * "query" : {
+        "metaInformation.externalId" : "9999999999"
+    },
+    "projection" : ["_id","entityType"]
+    }
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
+  * {
+    "message" : "List of entities fetched successfully",
+    "status" : 200,
+    "result" : [
+        {
+            "_id" : "5beaa888af0065f0e0a10515",
+            "entityType": "school"
+        }
+    ]
+  }
+  */
+
+    /**
+   * List of entities.
+   * @method
+   * @name find
+   * @param {Object} req - requested data.
+   * @param {String} req.params._id - requested entity type.         
+   * @returns {JSON} - Array of entities.
+   */
+
+  list(req) {
```

Comment:
@aman-tunerlabs - Make this API available by passing internal access token only.

Rename it to listByEntityIds

Query to be an array of entityIds and fields array required.

Reply:
@aks30  Entity ids should be external id or mongodb internal id.

Reply:
@priyanka-TL  Please make changes for request body 

{
  "entities" : [],
  "fields" : [] 
}.

Please do mongoid validation in request body .

Reply:
@aman-tunerlabs resolved changed req body and validation check